### PR TITLE
Fix paragraph.getRectsForRange crash on Windows

### PR DESCRIPTION
**Pull request by igordmn**
Fixes https://github.com/JetBrains/compose-jb/issues/1308

### CODE REVIEW

**Comment by VeselovAlex** [comment]
Should we add this workaround to `nativeJsMain`?

**Comment by igordmn** [comment]
I don't know. Was not able to run js tests:
```
Execution failed for task ':compileWasm'.
> Some files were not compiled. Check the log for more details
```

File: skiko/src/jvmMain/cpp/common/paragraph/Paragraph.cc [hunk 1]
```diff
@@ -86,6 +86,11 @@ extern "C" JNIEXPORT jobjectArray JNICALL Java_org_jetbrains_skia_paragraph_Para
     jobjectArray rectsArray = env->NewObjectArray((jsize) rects.size(), skija::paragraph::TextBox::cls, nullptr);
     for (int i = 0; i < rects.size(); ++i) {
         TextBox box = rects[i];
+        // TODO fix https://github.com/JetBrains/compose-jb/issues/1308 another way, we just masking the issue
+        // (but experiments show, that the result of GetRectsForRange is correct after that)
+        if (isnan(box.rect.fLeft) || isnan(box.rect.fTop) || isnan(box.rect.fRight) || isnan(box.rect.fBottom)) {
```
Comment (olonho) [inline skiko/src/jvmMain/cpp/common/paragraph/Paragraph.cc:91]: Why JVM-only, guess it shall be fixed on native as well.

Comment (igordmn) [reply skiko/src/jvmMain/cpp/common/paragraph/Paragraph.cc:91]: I have added for js too, but blindly, because I can't compile js.

**Review by VeselovAlex — APPROVED** [review]

**Review by olonho — APPROVED** [review]

**Pull request merged**